io.github.brainstormtrooper.facteur.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "io.github.brainstormtrooper.facteur",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.brainstormtrooper.facteur",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=ssh-auth",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home:create",
```

Comment:
why do you need `:create`.

Why do you need unfettered acccess to `home`?

Reply:
to save files created in the app

Reply:
if you use Gtk yu shouldn't have to with `GtkNativeFileDialog`.

Reply:
indeed...